easylinks/src/main/java/com/google/step/servlets/LoginServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.step.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  private static final UserService USERSERVICE = 
+      UserServiceFactory.getUserService();
```

Comment:
Maybe we could put the private variables at the end of the class?

Reply:
done